activities/normalize.go

```diff
@@ -0,0 +1,87 @@
+package activities
+
+import (
+	"context"
+
+	"github.com/bcc-code/bccm-flows/common"
+	"github.com/bcc-code/bccm-flows/services/ffmpeg"
+	"github.com/bcc-code/bccm-flows/services/transcode"
+	"go.temporal.io/sdk/activity"
+)
+
+type AnalyzeEBUR128Params struct {
+	FilePath       string
+	TargetLoudness float64
+}
+
+type AnalyzeEBUR128Result struct {
```

Comment:
Maybe reuse this result type (move it to common?), because it's duplicated three times now